{"name": "test_add2[【类型】输入特殊字符，给出提示信息1]", "status": "passed", "steps": [{"name": "步骤一：*&和0.2开始计算，预期应等于参数大小超出范围，打印<ExceptionInfo for raises contextmanager>", "status": "broken", "statusDetails": {"message": "TypeError: '>' not supported between instances of 'str' and 'int'\n", "trace": "  File \"D:\\zj\\PycharmProjects\\study202208\\study0818\\test\\test_calculate.py\", line 53, in test_add2\n    result=self.cal.add(x,y)\n  File \"D:\\zj\\PycharmProjects\\study202208\\script\\calculator.py\", line 4, in add\n    if a > 99 or a < -99 or b > 99 or b < -99:\n"}, "start": 1660806013560, "stop": 1660806013560}], "attachments": [{"name": "log", "source": "7ddb2967-ddd0-43d3-a3b4-b7f9364bccaf-attachment.txt", "type": "text/plain"}, {"name": "stdout", "source": "1012242f-d594-4562-b9e7-092a071e7681-attachment.txt", "type": "text/plain"}, {"name": "stderr", "source": "75bfc929-45ee-40b0-b300-3e1966a08a38-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "x", "value": "'*&'"}, {"name": "y", "value": "0.2"}, {"name": "expect", "value": "'参数大小超出范围'"}], "start": 1660806013560, "stop": 1660806013560, "uuid": "496185d4-1ce1-48e0-a4ec-18cc3ce8e5c2", "historyId": "14c868acb194d8087bf412026f7ce2df", "testCaseId": "c1fd4152c4894ab47d6f7448dc1c5bcf", "fullName": "study0818.test.test_calculate.TestAdd#test_add2", "labels": [{"name": "feature", "value": "加法计算模块"}, {"name": "story", "value": "加法报错用例"}, {"name": "tag", "value": "P2"}, {"name": "parentSuite", "value": "study0818.test"}, {"name": "suite", "value": "test_calculate"}, {"name": "subSuite", "value": "TestAdd"}, {"name": "host", "value": "LAPTOP-BEO6NH7O"}, {"name": "thread", "value": "21868-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "study0818.test.test_calculate"}]}